businesscentral-monitoring/image.yaml

```
@@ -1,20 +1,20 @@
 schema_version: 1
 
 name: "rhpam-7/rhpam-businesscentral-monitoring-rhel8"
-description: "Red Hat Business Central Monitoring 7.13 OpenShift container image"
+description: "IBM Business Automation Manager Open Editions Business Central Monitoring 7.13 OpenShift container image"
```

we could use only replacement of Red Hat with IBM i think.
Where is the full name use BAMOE but otherwise it is good to replace only necessary refs/names.

```suggestion
description: "IBM Business Central Monitoring 7.13 OpenShift container image"
```